src/ResourceManagement/Logic/Microsoft.Azure.Management.Logic/Properties/AssemblyInfo.cs

```diff
@@ -11,7 +11,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
-[assembly: AssemblyVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
Assembly file version should change.  Assembly Version should only change if there are breakign changes (and then it should change to 3.0)